packages/gatsby-theme-newrelic/src/components/Banner.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/core';
+import Button from './Button';
+import Icon from './Icon';
+
+const Banner = ({ children, color }) => {
```

Comment:
Could you talk me through the motivation behind making the `color` a prop? Are we planning to have multiple banners with different colors?

Reply:
we wanted to make it as easy as possible for the content creators to configure or for us to configure on the website, but what are your thoughts on this?

Reply:
I personally am hesitant to allow this flexibility because this allows someone to pass in arbitrary color styles that may or may not fit with the theme. I would rather start out a bit rigid and set the colors ourselves so we can maintain consistency. If we find we need this flexiblity, this would be a great addition in the future.

Reply:
A color prop is cool, but I'm thinking we may want to standardize on the color, so we don't get some weird banners being deployed. For our color palette it seems we may want to use the inverse of the header navigation color on each mode.

so if a user is on dark mode:  the banner color is: `background-color: #F4F5F5` and `font-color: #EDEEEE` and if the user is is on light mode:  the banner color is: `background-color: #22353C` and `font-color: #678082`

@djsauble  Thoughts?

Reply:
Agree we should be opinionated about this. In particular, the screenshots in the description look like the banner can exist without a background color, which makes the header nav look like it's floating.

I'm with @jerelmiller that we should use specific colors. Let's start with his suggestions.